patches/net/minecraft/client/resources/SplashManager.java.patch

```diff
@@ -0,0 +1,44 @@
+--- a/net/minecraft/client/resources/SplashManager.java
++++ b/net/minecraft/client/resources/SplashManager.java
+@@ -34,12 +_,12 @@
+ 
+     protected List<String> prepare(ResourceManager p_118869_, ProfilerFiller p_118870_) {
+         try {
+-            List list;
++            List<String> list;
+             try (BufferedReader bufferedreader = Minecraft.getInstance().getResourceManager().openAsReader(SPLASHES_LOCATION)) {
+                 list = bufferedreader.lines().map(String::trim).filter(p_118876_ -> p_118876_.hashCode() != 125780783).collect(Collectors.toList());
+             }
+ 
+-            return list;
++            return net.neoforged.neoforge.client.resources.NeoSplashHooks.prepare(list, p_118869_, p_118870_);
+         } catch (IOException ioexception) {
+             return Collections.emptyList();
+         }
+@@ -52,20 +_,12 @@
+ 
+     @Nullable
+     public SplashRenderer getSplash() {
```

Comment:
No reason to patch out all the vanilla code here. Inject the new hooks at the head of the method.